portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,20 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Erin!</h1>");
-  }
+    private List<String> names;    
+
+    @Override
+    public void init() {
+        names = new ArrayList<>();
```

Comment:
nit: consider declaring the arraylist with default values
eg.
`new ArrayList<String>(Arrays.asList("Erin", "Ruchi", "Owen"));`

Reply:
done.